docs/features.md

```diff
@@ -177,7 +213,7 @@ The document lists and describes all the feature flags that are available in Kym
   }
   ```
 
-- **KUBECONFIG_ID** – is used to configure the URL to which Busola sends a request to download a kubeconfig file. If you add `?kubeconfigID={your ID}` to the Busola URL, Busola tries to download the kubeconfig from `{kubeconfigUrl}/{yourID}`. If the operation succeeds, Busola adds the kubeconfing file to the cluster.
+* **KUBECONFIG_ID** – is used to configure the URL to which Busola sends a request to download a kubeconfig file. If you add `?kubeconfigID={your ID}` to the Busola URL, Busola tries to download the kubeconfig from `{kubeconfigUrl}/{yourID}`. If the operation succeeds, Busola adds the kubeconfing file to the cluster.
```

Comment:
Why those changes?

Reply:
autochanges made by prettify

Reply:
It looks like it does not affect the doc structure. Will it stay this way after the merge? :)

Reply:
it will. That's what prettify does. It has a set of own formatting rules (that can be adjusted to some extent), but it doesn't change how the code works